config/settings.yml

```diff
@@ -64,3 +64,5 @@ collections_to_show_attribution:
 # in shared_configs for embed-prod, we set this value to 0  
 analytics_debug: true
 min_files_to_search_default: 10
+enabled_features:
+  transcripts: true
```

Comment:
Please set the default to false. Then turn it on in any environment you want to see it in.

Reply:
Ok, it is set to false by default now. Maybe we don't need the shared_config changes now?